appcues/src/main/java/com/appcues/ui/primitive/OptionSelectPrimitive.kt

```diff
@@ -250,9 +250,16 @@ private fun List<OptionSelectPrimitive.OptionItem>.ComposePicker(
                     expanded = false
                     valueSelectionChanged(setOf(it.value))
                 }) {
-                    it.content.Compose()
+                    it.contentView(selectedValue == it.value).Compose()
```

Comment:
This is the picker expanded menu - if the item in the menu list is the selected item, it is highlighted using its selected style, if exists - again unsure if this is desired for our picker behavior but seems valid.